### PR TITLE
modify timing of logging serialFilter loaded message

### PR DESCRIPTION
**Pull request by toshiyamamoto**
The serialFilter loaded message needs to be logged after the server version is logged in console.log file.

### CODE REVIEW

**Comment by toshiyamamoto** [comment]
#build

**Comment by toshiyamamoto** [comment]
#build

**Comment by skasund** [comment]
L2 message review completed

**Comment by toshiyamamoto** [comment]
A PB looks good.

File: dev/com.ibm.ws.kernel.boot.nested/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ###############################################################################
-# Copyright (c) 2010, 2016 IBM Corporation and others.
+# Copyright (c) 2010, 2019 IBM Corporation and others.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.kernel.boot.nested/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: Ok

File: dev/com.ibm.ws.kernel.boot.nested/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops [hunk 2]
```diff
@@ -740,3 +740,5 @@ info.initlogs=Redirecting stdout and stderr to file {0}
 info.no.servers.defined=There are no servers defined in the user directory {0}.
 info.list.of.defined.servers=The following servers are defined relative to the user directory {0}.
 info.communicate.server=A communication error has occurred between the command {0} and the server {1}.
+
+info.serialFilterLoaded=Serial Filter Agent is loaded.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.kernel.boot.nested/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: Suggestion: The serial filter agent loaded.

File: dev/com.ibm.ws.kernel.boot.cmdline/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ###############################################################################
-# Copyright (c) 2010, 2016 IBM Corporation and others.
+# Copyright (c) 2010, 2019 IBM Corporation and others.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.kernel.boot.cmdline/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: Ok

File: dev/com.ibm.ws.kernel.boot.cmdline/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops [hunk 4]
```diff
@@ -748,3 +748,5 @@ info.initlogs=Redirecting stdout and stderr to file {0}
 info.no.servers.defined=There are no servers defined in the user directory {0}.
 info.list.of.defined.servers=The following servers are defined relative to the user directory {0}.
 info.communicate.server=A communication error has occurred between the command {0} and the server {1}.
+
+info.serialFilterLoaded=Serial Filter Agent is loaded.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.kernel.boot.cmdline/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: Suggestion: The serial filter agent loaded.

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ###############################################################################
-# Copyright (c) 2010, 2016 IBM Corporation and others.
+# Copyright (c) 2010, 2019 IBM Corporation and others.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: Ok

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops [hunk 6]
```diff
@@ -780,3 +780,4 @@ info.no.servers.defined=There are no servers defined in the user directory {0}.
 info.list.of.defined.servers=The following servers are defined relative to the user directory {0}.
 info.communicate.server=A communication error has occurred between the command {0} and the server {1}.
 
+info.serialFilterLoaded=Serial Filter Agent is loaded.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: Suggestion: The serial filter agent loaded.

File: dev/com.ibm.ws.kernel.boot.nested/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops (hunk 2 above)
Comment (toshiyamamoto) [reply dev/com.ibm.ws.kernel.boot.nested/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: Hi Charlotte, got it. I'll update the text. Thank you for your review.

Comment (Charlotte-Holt) [reply dev/com.ibm.ws.kernel.boot.nested/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: @toshiyamamoto Great! After the updates are made, just let me know, and I'll add the ID reviewed tag. Thanks!

**Comment by toshiyamamoto** [comment]
#build

File: dev/com.ibm.ws.kernel.boot.nested/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops (hunk 2 above)
Comment (toshiyamamoto) [reply dev/com.ibm.ws.kernel.boot.nested/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: I just updated the four files (I missed updating one files before..), and pushed. Thank you again.

**Pull request merged**